en/docs/setup/data-purging.md

```diff
@@ -237,9 +237,14 @@ You can perform data purging by clearing the session data using the script given
 
     !!! info
 
-        For DB2, MySQL, Oracle, and Postgre database scripts, see [Stored Procedures](https://github.com/wso2/carbon-identity-framework/tree/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures).
+        - For DB2, MySQL, Oracle, and Postgre database scripts, see [Stored Procedures](https://github.com/wso2/carbon-identity-framework/tree/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures).
+        - You can change the session cleanup task in the stored procedure according to your DB policies. You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the session created time or the session expiry time. By default, the session created time is used.
+             ``` 
```

Comment:
Shall we mention this example is to EXPIRY_TIME based session cleanup?
When I saw the code blog after the above point, I felt like the given code snippet is for default case

Reply:
fixed